pkg/client/helper.go

```diff
@@ -228,6 +230,12 @@ func TransportFor(config *Config) (http.RoundTripper, error) {
 		if tlsConfig != nil {
 			transport = &http.Transport{
 				TLSClientConfig: tlsConfig,
```

Comment:
It would probably be better to copy the http.DefaultTransform's fields (if these are truly the same) rather than add the new ones.